Source/Fuse.Navigation/TriggerActions.uno

```diff
@@ -115,10 +112,7 @@ namespace Fuse.Navigation
 
 		protected override void Perform(INavigation ctx, Node n)
 		{
-			var v = n as Visual;
-			if (v == null) return;
-
-			var page = Navigation.TryFindPage(Target ?? v);
+			var page = Navigation.TryFindPage((Node)Target ?? n);
```

Comment:
I think I only added it since it complained. The left/right sides need to be the same type I think.